packages/application/src/shell.ts

```diff
@@ -770,6 +774,31 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     }
   }
 
+  /**
+   * Activate Specified Area Widget.
+   */
+  activateWidget(area?: ILabShell.Area): void {
+    switch (area ?? 'main') {
+      case 'main':
+        const current = this._currentTabBar();
+        if (!current) {
+          return;
+        }
+        if (current.currentTitle) {
+          current.currentTitle.owner.activate();
+        }
+        return;
+      case 'left':
+      case 'right':
+      case 'header':
+      case 'top':
+      case 'menu':
+      case 'bottom':
+        return;
```

Comment:
Should it raise non implemented error or at least log a warning?

Reply:
I have thrown a not implemented error here, thinking about how to implement these